content/docs/setup-and-administration/configuring-app-registrations-for-non-interactive-use/index.md

```diff
@@ -0,0 +1,145 @@
+
+1. Define your new Application JSON and save it to a file locally. e.g.
+
+```json
+{
+    "display_name": "TrafficLight101",
+    "custom_claims": {
+      "serial_number": "TL1000000101",
+      "has_cyclist_light": "true"
+    }
+}
+```
+
+2. Generate a token using your pre-existing `App Registration` details
+
+```bash
+RESPONSE=$(curl \
+    https://app.rkvst.io/archivist/iam/v1/appidp/token \
+    --data-urlencode "grant_type=client_credentials" \
+    --data-urlencode "client_id=${CLIENTID}" \
+    --data-urlencode "client_secret=${SECRET}")
+
+TOKEN=$(echo -n $RESPONSE | jq .access_token | tr -d '"')
+```
+
+and save it locally to a bearer_token file.
+
+3. Submit your new Application JSON to the App Registration API Endpoint 
+
+```bash
+curl -X POST \
+     -H "@$BEARER_TOKEN_FILE" \
+     -H "Content-Type: application/json" \
+     -d "@/path/to/jsonfile" \
+     $URL/archivist/iam/v1/applications
+```
+
+You should see a response with details about the App Registration's `CLIENT ID` and `SECRET`:
+
+```json
+{
+    "identity": "applications/d1fb6c87-faa9-4d56-b2fd-a5b70a9af065",
+    "display_name": "TrafficLight101",
+    "client_id": "d1fb6c87-faa9-4d56-b2fd-a5b70a9af065",
+    "tenant_id": "tenant/53e6bed7-6f4c-4a37-8c4f-cf889f2b1aa6",
+    "credentials": [
+        {
+            "secret":"a0c09972b6ac912a4d67815fef88093c81a99b49977d35ecf6d162631aa29173",
+            "valid_from": "2021-09-21T16:43:19Z",
+            "valid_until": "2022-09-21T16:43:19Z"
+        }
+    ],
+    "custom_claims": {
+        "serial_number": "TL1000000101",
+        "has_cyclist_light": "true"
+    }
+}
+```
+
+{{< caution >}}
+**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later.
+{{< /caution >}}
+
+4. You should now have a newly configured App Registration and have recorded its `CLIENT_ID` and its `SECRET` so that it can be used to [generate a token](./#getting-a-token-with-your-app-registration) and [access the RKVST API](./#testing-your-access).
+
+For more details check out our [App Registrations API Reference](../../api-reference/app-registrations-api) which not only contains more detailed usage examples but also has the full OpenAPI Reference.
+
+## Getting a Token With Your App Registration
+
+Having completed the steps at [Creating an App Registration](./#creating-an-app-registration), and having taken note of the `CLIENT ID` and the `SECRET`, a token can be obtained with the following command.
+
+Replace `${CLIENTID}` with the application id, and `${SECRET}` with your secret from the application registration.
+
+```bash
+$ RESPONSE=$(curl \
+    https://app.rkvst.io/archivist/iam/v1/appidp/token \
+    --data-urlencode "grant_type=client_credentials" \
+    --data-urlencode "client_id=${CLIENTID}" \
+    --data-urlencode "client_secret=${SECRET}")
+
+$ TOKEN=$(echo -n $RESPONSE | jq .access_token | tr -d '"')
+```
+
+### Testing Your Access
```

Comment:
This doesn't test access, it just roughly checks if the tokens well. formed. Can you add a line to the effect of 'You can test the token with any of the kist APIs, for instance [link] fetching all assets'

Reply:
Should be resolved